content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -1,135 +1,290 @@
+like HTTP or HTTPS. Therefore, you will need to configure Blue Ocean's public
+SSH key with your Git server instead (as described
+<<remote-repositories,above>>), or by using one of the other hosted Git server
+options documented on this page.
+====
+
+[NOTE]
+====
+When Blue Ocean generates its own public SSH key for your current/logged in
+Jenkins user, Jenkins registers this credential with the following details for
+this Jenkins user:
+
+* *Domain*: `blueocean-private-key-domain`
+* *ID*: `jenkins-generated-ssh-key`
+* *Name*: `<jenkins-username> (jenkins-generated-ssh-key)`
+====
+
+
+=== From a repository on GitHub
+
+To create your Pipeline project directly from a repository on GitHub, click the
+*GitHub* button under *Where do you store your code?*
 
 [.boxshadow]
-image:blueocean/creating-pipelines/github-new-pipeline.png['Creating a Single
-Pipeline',width=60%]
+image:blueocean/creating-pipelines/where-do-you-store-your-code.png['Where do
+you store your code',width=70%]
 
-After selecting a repository Blue Ocean will scan all the branches in that Repository
-and will create a Pipeline for each branch containing a "Jenkinsfile" in the root folder.
-Blue Ocean will then run the Pipeline created for each branch in this process.
+In the *Connect to GitHub* section, enter your GitHub access token into the
+*Your GitHub access token* field. +
+If you previously configured Blue Ocean to connect to GitHub using a personal
+access token, Blue Ocean takes you directly to the
+<<choose-your-github-account-organization-and-repository,choosing your GitHub
+account/organization and repository>> steps below.
 
-If no branches in the selected repository have a "Jenkinsfile",
-Blue Ocean will offer to "Create a New Pipeline" for that repository,
-taking the user to the
-<<pipeline-editor#, Blue Ocean Pipeline Editor>>
-to create a new `Jenkinsfile` and add a new Pipeline based on that.
+[.boxshadow]
+image:blueocean/creating-pipelines/connect-to-github.png['Connect to
+GitHub',width=70%]
+
+If you do not have a GitHub access token, click the *Create an access key here*
+link to open GitHub to the *<<create-your-access-token,New personal access
+token>>* page.
+
+
+==== Create your access token
+
+. In the new tab, sign in to your GitHub account (if necessary) and on the
+  GitHub *New Personal Access Token* page, specify a brief *Token description*
+  for your GitHub access token (e.g. `Blue Ocean`). +
+  *Note:* An access token is usually an alphanumeric string that respresents
+  your GitHub account along with permissions to access various GitHub features
+  and areas through your GitHub account. The new access token process (triggered
+  through the *Create an access key here* link above) has the appropriate
+  permissions pre-selected, which Blue Ocean requires to access and interact
+  with your GitHub account.
+. Scroll down to the end of the page (optionally selecting additional scopes you
```

Comment:
"additional scopes you may require" is not really relevant, Jenkins currently cannot do anything more with those scopes even if you did add them :(

Reply:
No worries - I removed the bit in parentheses.